test/regression/cases/file_to_blackhole/experiment.yaml

```diff
@@ -1,5 +1,5 @@
 optimization_goal: cpu
-erratic: true
```

Comment:
Given that we don't fully understand the file rotation stuff, I'm not confident that the load we specify is properly being applied.

The CPU goal may be more stable in terms of results, but since the load application is still uncertain, I'd say leave this as erratic

Reply:
Addressed in c044814696